components/test-page-setup.js

```diff
@@ -122,7 +122,28 @@ function chartData(testState) {
         ]
     }
 }
-
+function chartDataEmpty(){
```

Comment:
if we only reset the chart when we start a new test run, I don't think we'll need this config

Reply:
as long as we want to restart the chart i think we need to give it an empty box or it will just shrink the histogram first then increase. I move the resetChart() to the onClick method and now it only clear after hitting start test